ombu_labs-auth.gemspec

```diff
@@ -23,4 +23,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "devise", "~> 4.8.1"
   spec.add_dependency "omniauth", "~> 2.1.0"
   spec.add_dependency "omniauth-github", "~> 2.0.0"
+  spec.add_dependency "omniauth-rails_csrf_protection"
+  spec.add_development_dependency "capybara"
+  spec.add_development_dependency "webdrivers"
+  spec.add_development_dependency "puma"
```

Comment:
@arielj Not sure if it is a good idea to add this here. It might be best to keep these dependencies in `Gemfile` -- It seems that dev dependencies could be removed from the gem spec file.

Reply:
I'm not sure about this, isn't this the place to add development dependencies? so it shows up in rubygems

maybe there's a rule on which dev dependencies goes in the gemspec and which goes in the Gemfile? I couldn't find anything clear on that topic

(I left the gems specified in the Gemfile because they were already there, but I don't know why they were not development dependencies)

Reply:
.